[2025-04-08 16:17:09,169] [ERROR] in app: Exception on /dashboard [GET]
Traceback (most recent call last):
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/routes.py", line 56, in dashboard
    return render_template('landing/dashboard.html', user=user)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 1295, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/dashboard.html", line 1, in top-level template code
    {% extends 'landing/base.html' %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/base.html", line 22, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/dashboard.html", line 50, in block 'content'
    <h5 class="card-title text-dark">{{ news['title'] }}</h5>
    ^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 471, in getitem
    return obj[argument]
           ~~~^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'news' is undefined
[2025-04-08 16:17:10,126] [ERROR] in app: Exception on /dashboard [GET]
Traceback (most recent call last):
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/routes.py", line 56, in dashboard
    return render_template('landing/dashboard.html', user=user)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 1295, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/dashboard.html", line 1, in top-level template code
    {% extends 'landing/base.html' %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/base.html", line 22, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/dashboard.html", line 50, in block 'content'
    <h5 class="card-title text-dark">{{ news['title'] }}</h5>
    ^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 471, in getitem
    return obj[argument]
           ~~~^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'news' is undefined
[2025-04-08 16:17:26,345] [ERROR] in app: Exception on /dashboard [GET]
Traceback (most recent call last):
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/routes.py", line 56, in dashboard
    return render_template('landing/dashboard.html', user=user)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 1295, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/dashboard.html", line 1, in top-level template code
    {% extends 'landing/base.html' %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/base.html", line 22, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/landing/templates/landing/dashboard.html", line 50, in block 'content'
    <h5 class="card-title text-dark"></h5>
    ^^^^^^^^^^^^^^^
  File "/Users/alessio/PYTHON_SOFTWARE/CMS_DEF/CMS_DEF/lib/python3.13/site-packages/jinja2/environment.py", line 471, in getitem
    return obj[argument]
           ~~~^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'news' is undefined
[2025-04-08 16:17:31,562] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:21:48,973] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:25:43,682] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:30:59,491] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:38:58,992] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:41:48,991] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:44:24,491] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:46:32,035] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:52:30,390] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 16:54:46,173] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:02:55,789] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:07:24,642] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:08:25,165] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:08:52,100] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:09:31,868] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:09:37,616] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:09:41,487] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:09:45,550] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:10:27,645] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:10:33,771] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:10:38,678] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:10:41,140] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:15:40,990] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:26:50,678] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:29:36,005] [INFO] in app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:34:27,468] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:35:56,223] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
[2025-04-08 17:36:32,968] [INFO] in landing_app: ✅ LinkBayCMS avviato correttamente.
